src/test/java/WeekdaysTest.java

```diff
@@ -0,0 +1,132 @@
+import org.junit.Assert;
+import org.junit.Test;
+
+public class WeekdaysTest {
+
+    @Test
+    public void shouldCheckWeekDayNameForDayNumber_1(){
+        //given
+        Weekdays weekdays = new Weekdays();
```

Comment:
ciekawostka :) Jeżeli mamy w wielu miejscach powtórzenie inicjalizacji obiektu, możemy to zrobić globalnie w klasie , będzie to wyglądać tak

```
public class WeekdaysTest {

    Weekdays weekdays = new Weekdays();

    //metody testów
}
```

wtedy nie musimy powtarzać tego kodu w każdym teście. Jest jeszcze jeden element jakiego możemy użyć, a mianowicie annotacji `@Before`

```
public class WeekdaysTest {

    Weekdays weekdays;

    @Before
    public void setup() {
        this.weekdays = new Weekdays();
    }
    
    //metody testów
}
```